bridge-token-factory/tests/token_transfer.rs

```diff
@@ -245,101 +327,199 @@ fn test_with_invalid_proof() {
     err_is(
-        &call_borsh!(user, factory_id.deposit(proof)),
+        &rt.block_on(
+            user.call(&worker, factory.id(), "deposit")
+                .max_gas()
+                .args(proof.try_to_vec().unwrap())
+                .transact(),
+        ),
         "Event cannot be reused for depositing.",
     );
 }
 
 #[test]
 fn test_bridge_token_failures() {
-    let (user, factory) = setup_token_factory();
-
-    call!(
-        user,
-        factory.deploy_bridge_token(DAI_ADDRESS.to_string()),
-        deposit = to_yocto("35")
-    )
-    .assert_success();
+    let (user, factory, worker) = create_contract();
+    let rt = Runtime::new().unwrap();
+
+    assert!(&rt
+        .block_on(
+            user.call(&worker, factory.id(), "deploy_bridge_token")
+                .deposit(parse_near!("35 N"))
+                .args(
+                    json!({"address": DAI_ADDRESS.to_string()})
+                        .to_string()
+                        .into_bytes()
+                )
+                .max_gas()
+                .transact()
+        )
+        .unwrap()
+        .is_success());
 
-    let token_account_id = format!("{}.{}", DAI_ADDRESS, FACTORY);
+    let token_account_id = format!("{}.{}", DAI_ADDRESS, factory.id());
 
     // Fail to withdraw because the account is not registered (and have no coins)
     err_is(
-        &call_json!(user, token_account_id.withdraw({
-                "amount" : "100",
-                "recipient" : SENDER_ADDRESS.to_string()}), deposit=1),
-        "The account is not registered",
+        &rt.block_on(
+            user.call(&worker, &token_account_id.parse().unwrap(), "withdraw")
+                .max_gas()
+                .deposit(1)
+                .args(
+                    json!({
+                        "amount" : "100",
+                        "recipient" : SENDER_ADDRESS.to_string()
+                    })
+                    .to_string()
+                    .into_bytes(),
+                )
+                .transact(),
+        ),
+        " is not registered",
     );
 
     // Register the account
-    let account_id: Option<ValidAccountId> = None;
+    let account_id: Option<AccountId> = None;
     let registration_only: Option<bool> = None;
-    call_json!(user, token_account_id.storage_deposit({
-        "account_id": account_id,
-        "registration_only": registration_only,
-    }))
-    .assert_success();
+
+    assert!(&rt
+        .block_on(
+            user.call(
+                &worker,
+                &token_account_id.parse().unwrap(),
+                "storage_deposit"
+            )
+            .max_gas()
+            .args(
+                json!({
+                    "account_id": account_id,
+                    "registration_only": registration_only
+                })
+                .to_string()
+                .into_bytes()
+            )
+            .deposit(parse_near!("1 N"))
```

Comment:
Where this deposit value is coming from?

Reply:
fixed: https://github.com/aurora-is-near/rainbow-token-connector/pull/159/commits/87ee9af208bad1f1faafcf4ba8e9aaf48e782e82

If deposit wasn't set the next const were used: https://docs.rs/near-sdk-sim/latest/near_sdk_sim/constant.STORAGE_AMOUNT.html